client/src/app/site/pages/meetings/pages/participants/pages/participant-detail/components/participant-detail-view/participant-detail-view.component.ts

```diff
@@ -147,8 +148,8 @@ export class ParticipantDetailViewComponent extends BaseMeetingComponent {
                 .subscribe(enabled => (this._isVoteDelegationEnabled = enabled))
         );
 
+        // TODO: Open groups subscription
```

Comment:
Is this todo still open or a reminder for later?

Reply:
This is a reminder. Currently groups need to be still fully subscribed within the active meeting subscription. Therefore a reminder.